core/modules/dashboard/includes/block.news.inc

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * @file
+ * Dashboard block displaying BackdropCMS.org news items.
+ */
+
+class DashboardNewsBlock extends Block {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getTitle() {
+    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('Backdrop News');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getAdminTitle() {
+    return t('Backdrop News');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getAdminPreview() {
+    return t('Displays a news feed from BackdropCMS.org.');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getContent() {
+    $feed = backdrop_http_request('https://backdropcms.org/api/notifications');
+    $links = array();
+    if ($feed->code === '200') {
+      $news = json_decode($feed->data);
+      foreach ($news->notifications as $key => $notifications) {
+        $links[] = '<p>' . l(
+            $notifications->notification->title,
+            $notifications->notification->url
+          ) . ' - ' . $notifications->notification->summary . '</p>';
```

Comment:
Although this is coming from backdropcms.org, we should do some basic filtering here. I suggest wrapping in a `filter_xss()` check.

Reply:
Done